packages/plasma-new-hope/src/components/Slider/components/SliderBase/SliderBase.styles.ts

```diff
@@ -42,11 +42,12 @@ export const RailWrap = styled.div`
 
 export const Fill = styled.div`
     position: absolute;
+    width: 0;
     height: 100%;
     top: 0;
     left: 0;
+    border-radius: var(--plasma-slider-rail-border-radius);
```

Comment:
Предлагаю поправить для единообразия
```suggestion
     border-radius: var(${tokens.railBorderRadius});
```